aca_preview/preview.py

```diff
@@ -80,17 +106,24 @@ def preview_load(load_name, outdir=None, report_level='none', loud=False):
     default is "none", meaning no reports are generated.  A final option is
     "all" which generates a report for every obsid.
 
-    :param load_name: Name of loads
-    :param outdir: Output directory
+    :param load_name: name of loads
+    :param outdir: output directory
     :param report_level: report level threshold for generating acq and guide report
-    :param loud: Print status information during checking
+    :param roll_level: level threshold for suggesting alternate rolls
+    :param loud: print status information during checking
+    :param acas: list of ACATable objects (optional, instead of ``load_name``)
+    :param obsids: list of obsids as str (optional)
```

Comment:
And guessing if acas is supplied that obsids must be as well?
```
--> 122         acas_dict = dict(zip(obsids, acas))
TypeError: zip argument #1 must support iteration
```

Reply:
Correct.

Reply:
Seems like it would make more sense to use the dict of acas instead of list if supplied, sort by time, and have obsids optional, but this is workable.

Reply:
Maybe, but I'm really ready to be done with this.

Reply:
Yep, I only mentioned it in case the idea of swapping it back sparked joy.

Reply:
You've been watching Marie Kondo.